src/Microsoft.SqlTools.ServiceLayer/ObjectManagement/ObjectTypes/User/UserHandler.cs

```diff
@@ -177,6 +177,30 @@ public override async Task<InitializeViewResult> InitializeObjectView(Contracts.
             }
             supportedUserTypes.Add(DatabaseUserType.NoLoginAccess);
 
+            string[] logins = DatabaseUtils.LoadSqlLogins(dataContainer.ServerConnection);
+
+            // If we couldn't load logins on current connection and this is a SQL DB connection 
+			// not to master then try to connect to master.  The "sys.sql_logins" DMV is not visible to user databases.  
+            if (logins.Length == 0 && isSqlAzure 
+                && string.Compare(parameters.Database, "master", true) != 0)
+            {
+                ServerConnection orgServerConnection = null;
```

Comment:
`orgServerConnection`, do you mean `masterDBConnection`?

Reply:
yeah, I was just connecting to the originally provided DB but later switched to use master to be more explict.  I'll change the name.